sdk/sphere/arm-sphere/recordings/node/sphere_test/recording_catalogs_create_test.json

```diff
@@ -9,9 +9,49 @@
         "Authorization": "Sanitized",
         "Connection": "keep-alive",
         "Content-Length": "21",
-        "Content-Type": "application/json; charset=UTF-8",
-        "User-Agent": "azsdk-js-arm-sphere-rest/1.0.0-beta.1 core-rest-pipeline/1.15.1 Node/18.19.0 OS/(x64-Windows_NT-10.0.22631)",
-        "x-ms-client-request-id": "d07daa7f-620d-4be6-97c3-725f292179c7"
+        "Content-Type": "application/json",
```

Comment:
RecorderError: Unable to find a record for the request PUT https://management.azure.com/subscriptions/azure_subscription_id/resourceGroups/myjstest/providers/Microsoft.AzureSphere/catalogs/resourcetest?api-version=2024-04-01